src/doodle/doodleCanvas.js

```diff
@@ -0,0 +1,83 @@
+import { render, createElement } from 'preact';
+import { useState, useEffect} from 'preact/hooks';
+import {Canvas} from './konvaFuncs';
+
+
+const DoodleCanvas = ({tool, size, active, attachedElement}) => {
+    //TODO: set up lines so that they scroll into and out of view using window scroll and offset coordinates
```

Comment:
Is this still a TODO? I think it should be working fine now with fixing the canvas size and not using position: fixed for a full-screen canvas.